src/Dash.NET/Dash.NET.fsproj

```diff
@@ -21,6 +21,32 @@
     <RepositoryType>git</RepositoryType>
     <FsDocsLicenseLink>https://github.com/plotly/Dash.NET/blob/dev/LICENSE</FsDocsLicenseLink>
     <FsDocsReleaseNotesLink>https://github.com/plotly/Dash.NET/blob/dev/RELEASE_NOTES.md</FsDocsReleaseNotesLink>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <FileVersion>1.0.0.0</FileVersion>
+    <Version>0.1.0-alpha10</Version>
+    <PackageReleaseNotes>[Add Suave web server support]()
```

Comment:
This is not necessary, it gets added via the release target

Reply:
https://github.com/plotly/Dash.NET/blob/f869c1082f15487b180b4bc4436c85bc1dd135ca/build.fsx#L200

Reply:
Same for version.

Reply:
Thanks. Updated.